model/common/src/icon4py/model/common/grid/gridfile.py

```diff
@@ -0,0 +1,306 @@
+# ICON4Py - ICON inspired code in Python and GT4Py
```

Comment:
I didn't look closely at this file as I assume it just contains stuff that was moved from grid_manager. Let me know if I should.

Reply:
No, it is exactly as you say, I extracted things from grid_manager.py to here in order to avoid circular imports.